.github/workflows/ci.yaml

```diff
@@ -75,7 +76,9 @@ jobs:
         id: cache_providers
         uses: actions/cache@v2
         with:
-          path: ./.terraform/plugins
+          path: |
+            ./.terraform/plugins
+            ./.terraform/providers
           key: ${{ runner.os }}-terraform-${{ matrix.tf_version }}-providers-${{ hashFiles('./versions.tf') }}
```

Comment:
What about .terraform.lock.hcl - maybe we could use it if it is available?

Reply:
We could, but it only applies to terraform 0.14+ and for earlier versions our `versions.tf` file already has all the versions locked, so the lockfile doesn't really add anything. I might just remove the `.terraform.lock.hcl` file to avoid confusion.

Reply:
Removed in b5380ed